client/post-editor/editor-title/index.jsx

```diff
@@ -52,40 +56,39 @@ export default React.createClass( {
 			const input = ReactDom.findDOMNode( this.refs.titleInput );
 			input.focus();
 		}
-	},
+	}
 
-	onChange( event ) {
+	onChange = event => {
 		if ( ! this.props.post ) {
 			return;
 		}
 
-		// TODO: REDUX - remove flux actions when whole post-editor is reduxified
-		PostActions.edit( {
+		this.props.editPost( this.props.site.ID, this.props.post.ID, {
 			title: event.target.value.replace( REGEXP_NEWLINES, ' ' )
 		} );
 		this.props.onChange( event );
```

Comment:
Another for potentially future pull request, doesn't appear we use this `event` object in the parent component, and if we ever were to, it might be better to pass the value directly rather than the DOM event object.